src/components/AppController.jsx

```diff
@@ -1,28 +1,28 @@
 import React from "react"
 import { Link } from "react-router"
-import FalcorController from "lib/falcor/FalcorController"
+import FalcorController from "../lib/falcor/FalcorController"
 import { setAppReady } from "lib/falcor/falcorUtils"
+import Navigation from '../components/Navigation';
 
 export default class AppController extends FalcorController {
   static getFalcorPath() {
     return ['appName']
   }
 
-  componentDidMount() {
+  handleMount() {
     super.componentDidMount()
     setAppReady()
   }
 
   render() {
     return (
       <div>
-        <h1>{this.state.data.appName}</h1>
-        <ul>
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/page/0">Page 0</Link></li>
-          <li><Link to="/page/1">Page 1</Link></li>
-          <li><Link to="/page/2">Page 2</Link></li>
-        </ul>
+        <div>
+          <Navigation
+            appName={this.state.data.appName}
+            onMount={this.handleMount()}
```

Comment:
Why are you passing the handleMount function to navigation? The only useful line, which is setAppReady, is supposed to be only rendered at the top-most component, which in this case is AppController.